### DISCUSSION TITLE
Not sure how to get pinned discussion to work

### DISCUSSION THREAD

**Discussion by martinamorris** [community/community#75] [Discussions] [answered]
When I choose the pin discussion feature, it pops up a window to format the post (nice!), but the only action I can see is to close that window -- there's no save feature, and the post does not get pinned.

I'm probably missing something simple...

**Comment by becca** [comment, marked as answer]
hey @martinamorris, super jazzed to hear you're trying to use the pinned discussions feature. i assume you have write+ permissions as you've been able to enable Discussions and access this feature so that sounds like it might be a 🐛 

there should be a `pin discussion` button:
<img width="934" alt="Screen Shot 2020-11-19 at 1 47 19 PM" src="https://user-images.githubusercontent.com/23545426/99716457-bfa51100-2a6d-11eb-928b-e550d19d028f.png">

can you please take a screenshot of what you're seeing?

**Reply by martinamorris** [reply]
Doh.  Sorry.

![image](https://user-images.githubusercontent.com/8194530/99717434-405c1100-2a5e-11eb-8924-ee03e892a1ae.png)

**Reply by becca** [reply]
oh yikes it looks like the button is there outside of the modal. i'll look into this. 
does clicking `pin discussion` work in this instance?

**Reply by martinamorris** [reply]
Yes!  I didn't even see the button down there.  But once I clicked, everything worked!
